pkg/security/tests/main_test.go

```diff
@@ -11,14 +11,76 @@ package tests
 import (
 	"flag"
 	"fmt"
+	"math/rand"
 	"os"
+	"strings"
 	"testing"
+	"time"
+
+	"github.com/cihub/seelog"
+	"golang.org/x/exp/slices"
 
 	"github.com/DataDog/datadog-agent/pkg/config/setup/constants"
 	"github.com/DataDog/datadog-agent/pkg/security/ptracer"
-	"golang.org/x/exp/slices"
+	"github.com/DataDog/datadog-agent/pkg/security/utils"
 )
 
+var (
+	testEnvironment  string
+	logLevelStr      string
+	logPatterns      stringSlice
+	logTags          stringSlice
+	logStatusMetrics bool
+	withProfile      bool
+	trace            bool
+	ebpfLessEnabled  bool
+)
+
+func SkipIfNotAvailable(t *testing.T) {
+	if ebpfLessEnabled {
+		available := []string{
+			"~TestProcess",
+			"~TestOpen",
+			"~TestUnlink",
+			"KillAction",
+			"TestRmdir",
+			"TestUnlink",
+			"TestRename",
+			"TestMkdir",
+			"TestUtimes",
+		}
```

Comment:
There's both `~TestUnlink` and `TestUnlink`. Should we remove `TestUnlink`  and only keep `~TestUnlink`?

Reply:
Good catch, thanks